fdb-relational-api/src/main/java/com/apple/foundationdb/relational/api/Options.java

```diff
@@ -188,6 +188,12 @@ public enum Name {
          * that can be used in EXECUTE CONTINUATION statements.
          */
         CONTINUATIONS_CONTAIN_COMPILED_STATEMENTS,
+
+        /**
```

Comment:
Could you document the scope of that option? 
Basically indicate if it is scoped to the query, the connection, the engine, etc. 
(See other options above)